### PR TITLE
Bump SRP packages to 13.1.2

### PR DESCRIPTION
**Pull request by theopnv**
### Purpose of this PR

---

### CODE REVIEW

**Comment by theo-pnv** [comment]
The 2 failing jobs are also failing on master (being discussed [here](https://unity.slack.com/archives/CFZERNTS7/p1636556165093800)). I checked the logs and there is no new failure that would be related to this PR.

File: com.unity.visualeffectgraph/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [13.1.2] - 2021-11-08
+## [13.1.2] - 2021-11-05
```
Comment (sebastienlagarde) [inline com.unity.visualeffectgraph/CHANGELOG.md:7]: note: weird that VFX was already 13.1.2?

**Review by sebastienlagarde — APPROVED** [review]

File: com.unity.visualeffectgraph/CHANGELOG.md (hunk 1 above)
Comment (theo-pnv) [reply com.unity.visualeffectgraph/CHANGELOG.md:7]: I think Ludovic bumped it in this PR already https://github.com/Unity-Technologies/Graphics/pull/6238. Not sure if that was done knowing the previous srp2core PR was already made but it's nice :)

**Review by esmelusina — APPROVED** [review]

**Comment by theo-pnv** [comment]
Merging now to avoid too merge conflicts on the changelogs.

**Pull request merged**